exporter/kafkaexporter/config_test.go

```diff
@@ -42,6 +44,20 @@ func TestLoadConfig(t *testing.T) {
 			NameVal: typeStr,
 			TypeVal: typeStr,
 		},
+		TimeoutSettings: exporterhelper.TimeoutSettings{
+			Timeout: 10 * time.Second,
+		},
+		RetrySettings: exporterhelper.RetrySettings{
+			Enabled:         true,
+			InitialInterval: 10 * time.Second,
+			MaxInterval:     1 * time.Minute,
+			MaxElapsedTime:  10 * time.Minute,
+		},
+		QueueSettings: exporterhelper.QueueSettings{
+			Enabled:      true,
+			NumConsumers: 2,
```

Comment:
Shouldn't this be one?

Reply:
Should it? it is 2 here because that's what I put in the yaml.

Reply:
Looks good I haven't noticed that this is a test :worried: